crates/rooch-framework/sources/auth_validator/bitcoin_validator.move

```diff
@@ -23,19 +23,24 @@ module rooch_framework::bitcoin_validator {
         BITCOIN_SCHEME
     }
 
+    /// `rotate_authentication_key_entry` only supports rotating authentication key to a Bitcoin legacy address
+    /// becuase ecdsa k1 scheme only supports key generation of 33-bytes compressed public key at this time.
```

Comment:
the generic type `<T>` should be removed.

Reply:
Done.